### PR TITLE
PKI State Proof Incremental Key Loading

### PR DESCRIPTION
**Pull request by winder**
## Summary

Followup to #3261 (contained in diff).

Use the new key loading routine from the REST API.

## Test Plan

New unit tests.

### CODE REVIEW

File: daemon/algod/api/algod.oas3.yml [hunk 1]
```diff
@@ -2817,6 +2837,93 @@
         "tags": [
           "private"
         ]
+      },
+      "post": {
+        "description": "Given a participation id, append state proof keys to a particular set of participation keys.",
```
Comment (tsachiherman) [inline daemon/algod/api/algod.oas3.yml:2842]: 1. `id` -> `ID`
2. drop the "." at the end of the sentence ( just to align with the rest of the descriptions )

**Review by tsachiherman — COMMENTED** [review]
overall looks good, I added few small requests.

File: daemon/algod/api/algod.oas3.yml (hunk 1 above)
Comment (winder) [reply daemon/algod/api/algod.oas3.yml]: Looks like the descriptions are almost equally divided! 158 with periods, 192 without. I'll save that exercise for another PR, there are probably other things that could make the verbiage more consistent in addition to punctuation.

File: daemon/algod/api/algod.oas3.yml [hunk 2]
```diff
@@ -2813,10 +2833,97 @@
             "description": "Unknown Error"
           }
         },
-        "summary": "Get participation key info by id",
+        "summary": "Get participation key info by ID",
```
Comment (tsachiherman) [inline daemon/algod/api/algod.oas3.yml:2836]: "Get participation key info by ID" -> "Get participation key info by participation ID"

**Comment by winder** [comment]
@tsachiherman this should be ready now

**Review by tsachiherman — APPROVED** [review]

**Pull request merged**